man/avocado.rst

```diff
@@ -1035,6 +1041,7 @@ The output should look like::
      (1/1) sleeptest.py:SleepTest.test: PASS (1.02 s)
     RESULTS    : PASS 1 | ERROR 0 | FAIL 0 | SKIP 0 | WARN 0 | INTERRUPT 0
     TESTS TIME : 1.02 s
+    JOB TIME   : 1.12 s
```

Comment:
Interesting that all the jobs were exactly of 0.1s longer than overall tests time 😄